archetypes/oci-mp/src/main/resources/README.md

```diff
@@ -0,0 +1,47 @@
+# Helidon MP OCI Sample Application
+
+This repository contains a Helidon MP sample application that integrates with the OCI SDK and demonstrates the following:
+
+1. Server code generation
+2. Client code generation
+3. Integration with the OCI SDK
+4. OCI IAM Authentication: User Principal and Instance Principal
+5. MicroProfile Metrics published to OCI Monitoring Service
+6. Logs published to OCI Logging Service
+7. OCI Custom Logs Monitoring using Unified Monitoring Agent
+8. Health and Liveness checks
+9. Configuration profiles for switching between `config_file` and `instance_principal` configurations
+
+This Project demonstrates API-driven development approach where the practice of designing and building APIs is done first, 
```

Comment:
I think it might be better to say

"This project demonstrates OpenApi-drive development" -> make project lowercase, and add "Open" before API

Reply:
Fixed